core/services/ardupilot_manager/mavlink_proxy/Manager.py

```diff
@@ -43,6 +43,9 @@ def set_master_endpoint(self, master: Endpoint) -> None:
     def start(self) -> None:
         self.tool.start(self.master)
 
+    def restart(self) -> None:
+        self.tool.restart()
```

Comment:
restart returns a bool to verify that the service was restarted, why are you ignoring it ?

Reply:
Did you see any parrots around?

Reply:
what ?

Reply:
I mean I don't know why am I ignoring it and I'm deliberately shifting your focus from that return.